docs/topics/sgx.md

```diff
@@ -3,15 +3,28 @@
 [Intel&reg; Secure Guard Extension](https://software.intel.com/en-us/sgx) (Intel&reg; SGX) is an architecture extension designed to increase the security of application code and data.
 Developers may choose [Intel&reg; SGX SDK](https://software.intel.com/en-us/sgx-sdk) or [Open Enclave SDK](https://github.com/Microsoft/openenclave/) to create applications that leverage this technology.
 
-Azure supports provisioning of SGX-enabled VMs under the umbrella of Azure Confidential Compute (ACC). You can create a Kubernetes cluster with one or multiple agent pool(s) running ACC VMs by specifying VM size `Standard_DC*` and distro `acc-16.04`.
+Azure supports provisioning of SGX-enabled VMs under the umbrella of Azure Confidential Compute (ACC). You can create a Kubernetes cluster with one or multiple agent pool(s) running ACC VMs by specifying a [DC-series](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general#dc-series) VM size with a supported distro from the table below.
+
+| OS           | distro      | Notes |
+| ------------ | ----------- |-------|
+| Ubuntu 16.04 | `acc-16.04` | specially built image with UEFI BIOS support
+| Ubuntu 18.04 | `aks-18.04` | standard Canonical Ubuntu image
```

Comment:
`aks-18.04` isn't the "standard Canonical Ubuntu image", it's a distro curated by AKS Engine with pre-installed components.

Reply:
Thank you @CecileRobertMichon . Right, I noticed `microsoft-aks` image publisher.
I will replace this line with `standard AKS image for Ubuntu 18.04`.
Would that be more descriptive?